heartbeat/galera

```diff
@@ -208,13 +208,30 @@ The galera cluster address. This takes the form of:
 gcomm://node,node,node
 
 Only nodes present in this node list will be allowed to start a galera instance.
-It is expected that the galera node names listed in this address match valid
-pacemaker node names.
+The galera node names listed in this address are expected to match valid
+pacemaker node names. If both names need to differ, you must provide a
+mapping in option pcmk_host_map.
```

Comment:
Aside from the concept, the pcmk_ prefix is (or should be) reserved.
Even if its the same kind of functionality, pick a different variable name :-)